app/views/users/new.html.erb

```diff
@@ -0,0 +1,40 @@
+<!--<form class="new_user" id="new_user" action="/users" accept-charset="UTF-8" method="post">-->
+<!--  <input type="hidden" name="authenticity_token" value="<%#= form_authenticity_token %>">-->
+
+<!--  <label for="user_name">Name</label>-->
+<!--  <input class="form-control" type="text" name="user[name]" id="user_name">-->
+
+<!--  <label for="user_email">Email</label>-->
+<!--  <input class="form-control" type="email" name="user[email]" id="user_email">-->
+
+<!--  <label for="user_password">Password</label>-->
+<!--  <input class="form-control" type="password" name="user[password]" id="user_password">-->
+
+<!--  <input type="submit" name="commit" value="Create my account" class="btn btn-primary" data-disable-with="Create my account">-->
+<!--</form>-->
+
+<%
+=begin%>
+ <%= form_with url: users_path do |f| %>
+  <%= f.label :name, "Name:" %>
+  <%= f.text_field :name %>
+
+  <%= f.label :email, "Email:" %>
+  <%= f.email_field :email %>
+
+  <%= f.label :password, "Password:" %>
+  <%= f.password_field :password %>
+
+  <%= f.submit "Submit" %>
+<% end %> 
+<%
+=end%>
+
+<%= form_with model: @user do |f| %>

```

Comment:
![image](https://user-images.githubusercontent.com/9512604/67961390-68ba6880-fbda-11e9-80ce-bd21025e3254.png)

Reply:
https://edgeguides.rubyonrails.org/form_helpers.html#using-form-for-and-form-tag

Reply:
Yeah, that why i said am aware, but sometimes we need to do things the old way and also we should learn how to follow/do the requirement of a project, so that even if you work on a old code base for a company you will still understand.
That all from me guys....